pkg/jx/cmd/create_vault.go

```diff
@@ -284,10 +290,10 @@ type kmsConfig struct {
 	project  string
 }
 
-func (o *CreateVaultOptions) createKmsConfig(vaultName string) (*kmsConfig, error) {
+func (o *CreateVaultOptions) createKmsConfig(vaultName string, clusterName string) (*kmsConfig, error) {
 	config := &kmsConfig{
-		keyring:  fmt.Sprintf("%s-keyring", vaultName),
-		key:      fmt.Sprintf("%s-key", vaultName),
+		keyring:  gke.VaultKeyringName(vaultName, clusterName),
```

Comment:
should this be tied to GKE here?  others that use `GKE` have `GKE` in the function name so it is obvious

Reply:
So all of these required a fairly hefty refactor moving all these methods into the `/cloud/gke` package under the new subpackage of `/cloud/gke/vault`, but that's done